src/test/regress/expected/multi_metadata_sync.out

```diff
@@ -876,8 +876,8 @@ WHERE
 	OR logicalrelid = 'mx_colocation_test_2'::regclass;
```

Comment:
Ideally this line should have an ORDER by so that the output doesn't change but if it is not flaky we can keep it the way it is.